.vib/gotrue/goss/vars.yaml

```diff
@@ -0,0 +1,6 @@
+binaries:
+  - gotrue
+files:
+  - paths:
+      - /opt/bitnami/gotrue/schema.sql
```

Comment:
I'm not sure we should be checking this file considering it is an upstream file and does not really depend on us.

I would instead suggest implementing a `command` test to check something like `find /opt/bitnami/gotrue -name "*.sql" | wc -l` is not 0 to ensure 'migrations' are present.